### PR TITLE
withdraw-by-check error docs

### PR DESCRIPTION
**Pull request by jrobinson01**
adds errors for "missing address" and "invalid loan/share"
@tkainz I left the logging message the same as the other errors but feel free to make suggestions. I believe these errors can occur in both the `STATESTART` and `PERFORMWITHDRAW` states.

### CODE REVIEW

**Comment by tkainz** [comment]
@jrobinson01  - Currently have the program coded (logging message) for the following error conditions:
"Account Warning Found"
"Invalid Share Type"
"Invalid Loan Type"
"Share Warning Found"
"Loan Warning Found"
"Invalid Address"
And the actual error thrown by the attempted open or read of the config file.

**Comment by jrobinson01** [comment]
Looks good to me @tkainz . You're clear on what error codes to map those to?

**Comment by tkainz** [comment]
@jrobinson01 - Just following the Yellow Brick.....I mean the JSON Contract!

File: withdraw-by-check/withdraw-by-check-json-contract.md [hunk 1]
```diff
@@ -108,10 +108,30 @@ If the request is not successful due to insufficient funds, the poweron should r
 }
 ```
 
+If the request is not successful due to a missing address, the poweron should respond with:
+
+```json
+{
+  "errorCode": 502,
+  "loggingErrorMessage": "TRANPERFORM Error:+TRANERROR"
```
Comment (tkainz) [inline withdraw-by-check/withdraw-by-check-json-contract.md]: Tranperform error?

Comment (jrobinson01) [reply withdraw-by-check/withdraw-by-check-json-contract.md]: I think that's what Christina originally sent over. I'm going to leave it to you to fill that in with whatever makes sense. We can do something simple like use the strings you mentioned https://github.com/Banno/banno-powerons/pull/8#issuecomment-593438132 or add to them if there's anything else you think would be useful.

File: withdraw-by-check/withdraw-by-check-json-contract.md [hunk 2]
```diff
@@ -108,10 +108,30 @@ If the request is not successful due to insufficient funds, the poweron should r
...
+}
```
Comment (tkainz) [inline withdraw-by-check/withdraw-by-check-json-contract.md]: How is this error code situation (503) different than the error code situation described on line 55?

Comment (jrobinson01) [reply withdraw-by-check/withdraw-by-check-json-contract.md]: Good catch. Looks like a duplicate. I'll adjust.

**Pull request merged**